meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -373,7 +373,9 @@ OSTREE_BOOTLOADER:imx8mm-lpddr4-evk-ebbr ?= "grub"
 IMAGE_EFI_BOOT_FILES:sota:imx8mm-lpddr4-evk-ebbr ?= "${@make_efi_dtb_boot_files(d)}"
 WKS_FILE:sota:imx8mm-lpddr4-evk-ebbr ?= "efidisk-sota.wks.in"
 # iMX8MM EVKB revision
-MACHINE_FEATURES:append:imx8mm-lpddr4-evk = " mxm-mwifiex-load"
+KERNEL_MODULE_AUTOLOAD:append:imx8mm-lpddr4-evk = " moal"
+KERNEL_MODULE_PROBECONF:append:imx8mm-lpddr4-evk = " moal"
+module_conf_moal:imx8mm-lpddr4-evk = "options moal mod_para=nxp/wifi_mod_para.conf drvdbg=0x6"
```

Comment:
Would be good to confirm indeed if this customization for mod_para is still needed as it seems it is not required anymore by looking at the BSP (maybe the default path it looks now is nxp/wifi_mod_para.conf?).

Reply:
And also if autoloading is still required as well, @MrCry0 should be able to confirm.